lib/marked.esm.mjs

```diff
@@ -0,0 +1,2206 @@
+/**
```

Comment:
I don't know what this project's processes are, but I'd imagine this file should only be updated when a release is cut? If this file is going to be added then the old `lib/marked.esm.js` should be removed